v7/src/v7_encode_api.c

```diff
@@ -113,8 +111,11 @@ int v7_block_encode_pri(bplib_mpool_bblock_primary_t *cpb)
 {
     v7_encode_state_t         v7_state;
     bplib_mpool_stream_t      mps;
-    CborEncoder               top_level_enc;
+    QCBOREncodeContext        top_level_enc;
     const bp_primary_block_t *pri;
+    MakeUsefulBufOnStack(scratch_buf, 256);
```

Comment:
What's 256 represent? Should it be a macro of some sort?

Reply:
Size of the buffer.

Reply:
I'd recommend putting that value in a macro like BPLIB_STANDARD_BUF_LENGTH, something like that, so it can be changed more easily. I see it referenced in other places too